OpenRA.Mods.Common/Util.cs

```diff
@@ -209,6 +209,31 @@ public static int RandomDelay(World world, int[] range)
 			return world.SharedRandom.Next(range[0], range[1]);
 		}
 
+		public static WDist RandomDistance(World world, WDist[] distance)
```

Comment:
I think the various `Random` utils should take `MersenneTwister` as a parameter so the caller can choose the RNG to poll. I realise `RandomDelay` is setting a bad example here, that would be worth fixing.

Reply:
It was already refactored to `Util.RandomInRange`. This pull request is just old.